pkg/epp/handlers/response.go

```diff
@@ -29,10 +29,7 @@ import (
 	logutil "sigs.k8s.io/gateway-api-inference-extension/pkg/epp/util/logging"
 )
 
-const (
```

Comment:
The reason for using `data: [DONE]` as keyword to detect ending message is that vLLM will return both:
```
data: {"id":"cmpl-36294ed39f844e5e951bb0bccad780a1","created":1735927823,"model":"meta-llama/Llama-2-7b-hf","choices":[],"usage":{"prompt_tokens":8,"total_tokens":13,"completion_tokens":5}}Elapsed 0.3072531223297119 seconds 

data: [DONE]Elapsed 0.30726003646850586 seconds 
```

in a same packet for streaming mode. Can you test the PR in a cluster to verify the change?

Reply:
Can you provide an example of the request you use and also the epp version and config? 

I tried the request I shared above and the token usage metrics works for me using the latest image from main branch.

Reply:
I think the issue is that the usage is set in every block, so we have accumulate them: https://github.com/kubernetes-sigs/gateway-api-inference-extension/pull/1627/files